Accept empty modules

For the Xen platform the all_video.mod module is empty.
With old binutils the .symtab section remained (containing
only section symbols), so the check didn't trigger, but starting
with binutils 2.27 not even a .symtab remains.  As there are
also no relocations that's no problem (and that is checked
independendly).
Index: grub-2.02~rc1/util/grub-module-verifierXX.c
===================================================================
--- grub-2.02~rc1.orig/util/grub-module-verifierXX.c
+++ grub-2.02~rc1/util/grub-module-verifierXX.c
@@ -224,7 +224,8 @@ check_symbols (const struct grub_module_
       s = find_section (arch, e, ".moddeps");
 
       if (!s)
-	grub_util_error ("no symbol table and no .moddeps section");
+	/*grub_util_error ("no symbol table and no .moddeps section");*/
+	return; /* An empty module happens for all_video.module for Xen */
 
       if (!s->sh_size)
 	grub_util_error ("no symbol table and empty .moddeps section");
